lib/wechat_pay/api/api.ex

```diff
@@ -117,15 +117,20 @@ defmodule WechatPay.API do
   @spec refund(map, Config.t()) ::
           {:ok, map} | {:error, WechatPay.Error.t() | HTTPoison.Error.t()}
   def refund(attrs, config) do
-    opts = [
-      ssl: [
-        cacerts: [config.ssl_cacert |> decode_public()],
-        cert: config.ssl_cert |> decode_public(),
-        key: config.ssl_key |> decode_private()
-      ]
-    ]
-
-    Client.post("secapi/pay/refund", attrs, opts, config)
+    ssl =
+      case config.ssl_cacert do
+        nil ->
+          [cert: config.ssl_cert |> decode_public(), key: config.ssl_key |> decode_private()]
```

Comment:
Multiline here would be appreciated. Thanks.

Reply:
OK.

Reply:
👍